### PR TITLE
#157387545: display free slots on schedule

### PR DESCRIPTION
**Pull request by Noblemajesty**
#### What does this PR do?
- adds available time slots on the event card
#### Description of Task to be completed?
- [X] write logic to determine free slot
- [x] set color indicator on free time to green
#### How should this be manually tested?
- clone repo and open in android studio
- sync and launch app for testing
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- story type - feature
- story title - Display free slots on the schedule
- story id - #157387545
#### Screenshots (if appropriate)
ezgif com-video-to-gif 2
screen shot 2018-05-16 at 5 10 42 pm

### CODE REVIEW

File: app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java [hunk 1]
```diff
@@ -91,7 +91,8 @@ public MakeGoogleCalendarCallPresenter(GoogleAccountCredential credential,
         List<CalendarEvent> calendarEvents = new ArrayList<>();
         DateTime now = new DateTime(System.currentTimeMillis());
         Events events = mService.events().list("primary")
-                .setMaxResults(6)
+
+                .setMaxResults(10)
```
Comment (TheDancerCodes) [inline app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java]: @Noblemajesty @pawnjester 
Is there a reason why the maximum number of results are capped at 10?

Comment (Noblemajesty) [reply app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java]: there's no reason @TheDancerCodes

Comment (TheDancerCodes) [reply app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java]: Hmmm...

I am trying to understand the max number of results.

I am assuming that these results are maximum number of events that can be displayed in our app from the calendar api, correct?
@Noblemajesty @pawnjester

Comment (Noblemajesty) [reply app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java]: @TheDancerCodes Yes, it is the maximum number of events returned on one result page. According to the documentation, the default value is 250. That is, 250 events at once.

Comment (TheDancerCodes) [reply app/src/main/java/com/andela/mrm/presenter/MakeGoogleCalendarCallPresenter.java]: Alright, thanks for the clarification.

**Comment by TheDancerCodes** [comment]
LGTM!

Probably 2 things we will need to consider in another story:
1) I see that you are using GMT+1. This must be West African Time. 
What about when the app is used in a different timezone? Is there a way we can abstract some functionality for picking the right timezone of the device?

2) The Max Results seem to be set to 10. What happens when the max results for a day are more than 10?
@Noblemajesty

**Pull request merged**